pages/Frequency/Identity.md

```diff
@@ -1,43 +1,51 @@
-# Identity
-
-## Purpose
-
-1. Specify how DSNP Identifiers map to Frequency.
-1. Specify how the DSNP delegation maps to Frequency.
-1. Detail how Frequency matches the ownership requirements of DSNP.
-
-## Details
-
-DSNP uses [Message Source Accounts (MSAs)](https://libertydsnp.github.io/frequency/pallet_msa/index.html) that map directly to a DSNP Identity.
-
-## Identifier
-
-MSAs each receive a unique unsigned 64-bit identifier that is used as the DSNP Id.
-These MSAs are pseudo-anonymous identifiers that are associated with one or more public keys.
-MSAs can be created by the user or on behalf of the user using a signed authorization.
-
-### Creating a DSNP User Id
-
-1. Generate a [compatible key pair](https://wiki.polkadot.network/docs/learn-keys) in your wallet.
-1. Create a Message Source Account (MSA) on Frequency using one of the `create` extrinsics.
-1. Wait for the transaction to process.
-
-### Retrieving a DSNP User Id from a Public Key
-
-1. Use the Frequency state query `msa.messageSourceIdOf` to retrieve the MSA Id from the public key generated in step 1.
-1. The MSA Id is the DSNP Id and can be used to generate the [DSNP User URI](../DSNP/Identifiers.md#dsnp-user-uri).
-
-## Ownership
-
-Frequency has a strong ownership model for MSAs.
-Ownership of an MSA is controlled by the associated keys.
-MSAs are the source for [Messages](https://libertydsnp.github.io/frequency/pallet_messages/index.html) either directly or via delegation.
+# Frequency Identity
+
+## Identity
+- **Name**: Message Source Account or MSA
+- **Representation**:
+The following data that constitute a Message Source Account are stored in the MSA pallet:
+  * The DSNP Id associated with this MSA
+  * Delegation relationships to Providers
+  * Schema permissions granted to Providers
+  * MSA user state (Profile, Graph, etc...)
+  * [Control keys](#control-keys)
+
+## DSNP Identifier
+- **Name**: Message Source Account Identifier, or MSA Id.
+- **Data Type**: `uint64`
+- **Mapping**: The MSA Id is able to be used directly as the DSNP Id
+- **Description**:
+At least one public key MUST be associated with an MSA Id for it to be considered active.
+
+## Control Keys
+- **Name**: `public_key`
+- **Data Type**: `AccountId`, Schnorrkel/Ristretto x25519 ("sr25519") derived cryptographic public key
+- **Description**: See [Cryptography on Polkadot](https://wiki.polkadot.network/docs/learn-cryptography).
+A public key CANNOT be associated with more than one MSA at a time.
 
 ## Delegation
+- **Name**: `delegation`
+- **Representation**:
+  The following data storage relates necessary information for retrieving and validating delegations:
+    * Provider registry (`pallet_msa`)
```

Comment:
If these are all part of `pallet_msa` maybe it can be referenced once separately from the individual items.